### DISCUSSION TITLE
Broken Discord link?

### DISCUSSION THREAD

**Discussion by nsbradford** [supabase/supabase#20776] [Questions]
The discord link at https://supabase.com/support appears to be broken? Is anyone else experiencing this?

**Comment by GaryAustin1** [comment]
What issue are you having?  It takes me to Discord but to a closed thread.  The current main thread is help-and-questions. Which is on the side panel.

**Comment by nsbradford** [comment]
<img width="1279" alt="image" src="https://github.com/supabase/supabase/assets/6633811/18c8b72b-c47b-40e8-9b84-996931b9fd46">

**Reply by GaryAustin1** [reply]
I'll mention this in Discord maybe a moderator there knows.  I can't test as it sends me directly in to discord as it knows I have an account.

**Comment by nsbradford** [comment]
working for me now, perhaps some transient thing - thanks 🙏
